content/articles/build-a-weather-forecast-app-with-next-js/index.md

```diff
@@ -0,0 +1,539 @@
+# Build  a weather forecast app in Next js
+
+
+## Introduction
+In this tutorial, I'll show you how to make a weather app in Next js. Showing you how to grab live web data from cities all across the world via an external API. We'll also use the API to look up the daily weather forecast.
+
+The API we’ll be using for this tutorial is openweather API but to do that we’ll have to sign up with them to be able to make use of their API.
+
+It's a really basic beginner project, and it's for those who have a basic understanding of React and want to learn how to make a server-side rendered react app with Next js. The only requirement is that you have a basic understanding of React. We’ll be using for this tutorial is openweather API but to do that we’ll have to signup with them to be able to make use of their API.
+
+At the end of this tutorial the readers should be able to:
+- know what next js is
+- when to use it
+- Know the key advantages next js has over React
+- Install next js
+- Be able to create an application in Next js
+- fetch an external API in next
+
+So sit back, relax and enjoy the ride!!!
+### OVERVIEW
+- Introduction
+- What is Next js
+- Why you should use it
+- Comparing Next with React
+- Building a weather forecast app 
+
+## What is Next js?
```

Comment:
You can make it Next.js or NextJS.

No spaces.

Do this change throughout the article.